app_flutter/test/service/google_authentication_test.dart

```diff
@@ -21,7 +21,8 @@ void main() {
           .thenAnswer((_) => const Stream<GoogleSignInAccount>.empty());
       when(mockSignIn.isSignedIn())
           .thenAnswer((_) => Future<bool>.value(false));
-      authService = GoogleSignInService(googleSignIn: mockSignIn);
+      authService = GoogleSignInService(googleSignIn: mockSignIn)
+        ..notifyListeners = () => null;
```

Comment:
I don't understand why notifyListeners needs to be set.

Reply:
Good point! This needs to be set for the id token test where we check this new functionality. I moved this line out of the general set up and to the specific tests that need it. It's use is for updating the dashboard with the latest state. In unit tests, it just needs to be stubbed out.